### ISSUE TITLE
Migrating over to Gamocosm

### ISSUE THREAD

**Issue by liangmason** [Gamocosm/Gamocosm#88]
Hello,

So I was trying to migrate my home hosted Enigmatica2 server to Gamocosm. I deleted ~minecraft and copied my minecraft folder to ~, renamed forge-version-universal.jar to minecraft_server-run.jar. 

Was there a step I missed, because whenever I start Minecraft on the site, I get a "Unable to start Minecraft: MCSW API error: action start response status not OK, was no_minecraft, {"pid"=>0, "status"=>"no_minecraft"}"

I also tried renaming my start script to minecraft_server-run.sh, no luck either.

**Comment by liangmason** [comment]
Ahh fixed

**Comment by mountain01** [comment]
I know this is closed @liangmason, but how did you fix your problem? I'm running into the same issue.

**Comment by Raekye** [comment]
Pretty late 😅 but the steps he described are generally correct. I'm not sure what the issue was. I'm guessing you've already moved on, but for anyone else having the same issue, please provide more details

**Comment by mountain01** [comment]
For anyone else running into this issue I'll give some more details about my situation and what I did to fix it.

I was trying to set up a Sky Factory 4 server on Gamocosm and I followed all the recommended steps.  I got the server files, uploaded them to my droplet after letting Gamocosm spin it up.  I renamed the startup sh script like I was supposed to. And then it appeared to not work,  Gamocosm couldn't start up my server.  

I then checked the script itself and found that it was trying to reference a .jar file that didn't exist.  It was then that I realized that I hadn't run the forge installer. I copied over the file, but never actually installed it. After running the install with `java -jar <forge-installer>` everything worked wonderfully.

Hopefully this helps others that try looking into a closed issue like I did.
